pkg/tasks/reset.go

```diff
@@ -29,6 +31,91 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
+// CleanupLoadBalancers Deletes all the load balancers services from the cluster.
+func CleanupLoadBalancers(s *state.State) error {
+	if !s.CleanupLoadBalancers {
+		return nil
+	}
```

Comment:
We should make this task conditional instead, see an example here https://github.com/kubermatic/kubeone/blob/8b8d79407b2ea755572b05042c152e83255bc590/pkg/tasks/tasks.go#L603